onnxruntime/test/testdata/kernel_def_hashes/arm_ops.cpu.json

```diff
@@ -0,0 +1,14 @@
+[
```

Comment:
These are arm only kernels, will this have conversion issue if we try to convert ONNX->ORT on intel hardware, where you may not be able to create inference sessions because of the missing kernel for s8s8 ops

Reply:
Thanks! Will take care of it in following PR.